### ISSUE TITLE
Publish pre-built binary for linux/arm64 (aarch64)

### ISSUE THREAD

**Issue by nacardin** [rustwasm/wasm-pack#1064]
#### 💻 Example
I would like to be able to run the install script on arm64 linux machines.
```
$ curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh 
```
Currently it fails with
```
wasm-pack-init: no precompiled binaries available for CPU architecture: aarch64
```

#### 💡 Description
* The release workflow should build and publish a binary for `aarch64-unknown-linux-musl`.
* The install script should look up and install pre-built binary for aarch64.

**Comment by Serveny** [comment]
Use `cargo install wasm-pack` instead

**Comment by d3lm** [comment]
Yep, using `cargo install` works fine.

**Comment by drager** [comment]
Fixed in [v0.11.0](https://github.com/rustwasm/wasm-pack/releases/tag/v0.11.0).
